<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportsDialog</class>
 <widget class="QDialog" name="ReportsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reports</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Dealer</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="dealer_comboBox">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(64, 66, 68);
color: rgb(146, 189, 108);
font: 14pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dealerfrom_dateEdit">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(64, 66, 68);
color: rgb(146, 189, 108);
font: 14pt &quot;Sans Serif&quot;;</string>
             </property>
             <property name="displayFormat">
              <string>dd-MM-yyyy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dealerto_dateEdit">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(64, 66, 68);
color: rgb(146, 189, 108);
font: 14pt &quot;Sans Serif&quot;;</string>
             </property>
             <property name="displayFormat">
              <string>dd-MM-yyyy</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QToolButton" name="printDealerPayment_toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="dealer_tableview"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Supplier</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="supplier_comboBox_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(64, 66, 68);
color: rgb(146, 189, 108);
font: 14pt &quot;Sans Serif&quot;;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="supplierfrom_dateEdit_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(64, 66, 68);
color: rgb(146, 189, 108);
font: 14pt &quot;Sans Serif&quot;;</string>
             </property>
             <property name="displayFormat">
              <string>dd-MM-yyyy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="supplierto_dateEdit_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(64, 66, 68);
color: rgb(146, 189, 108);
font: 14pt &quot;Sans Serif&quot;;</string>
             </property>
             <property name="displayFormat">
              <string>dd-MM-yyyy</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QToolButton" name="printsupplierpayments_toolbutton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="supplier_tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
